packages/create-sitecore-jss/src/templates/nextjs/src/byoc/index.client.tsx

```diff
@@ -16,4 +18,6 @@ import '@sitecore/components/form';
  */
 
 // An important boilerplate component that prevents BYOC components from being optimized away. Should be kept in this file.
-export default (props: unknown) => FEAAS.ExternalComponent(props);
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const DefaultExport = (props: ComponentProps) => FEAAS.ExternalComponent(props) || <></>;
```

Comment:
Curious, why the need to fallback to `<></>` here? In what scenario would `FEAAS.ExternalComponent` return falsy value?

Reply:
FEAAS.ExternalComponent return type has been changed from Component to Component  | undefined. This causes typescript errors in index.ts when bundle is loaded with dynamic() call, and this is a hack to fix that.

Reply:
Got it. Maybe a question for @Inviz then